packages/mobile/src/components/audio/Audio.tsx

```diff
@@ -368,9 +226,9 @@ export const Audio = () => {
         // If we aren't casting, update the progress
         global.progress = progress
       } else {
-        // If we are casting, only update the seekableDuration
+        // If we are casting, only update the duration
         // The currentTime is set via the effect in GoogleCast.tsx
-        global.progress.seekableDuration = progress.seekableDuration
+        global.progress.duration = progress.duration
```

Comment:
When we do the second pass at this, we should be able to remove the global and use the progress hook everywhere :)

Reply:
PLEASE